terraform/concourse/concourse.tf

```diff
@@ -159,7 +159,9 @@ resource "terraform_data" "provision_concourse" {
     command = <<-CMD
       gcloud compute ssh ${var.hostname} --zone ${var.zone} --tunnel-through-iap --command "\
       sudo /concourse/scripts/install.sh ${var.docker_ver} ${var.data_disk} ;\
-      sudo /concourse/scripts/start.sh ${var.hostname}.${var.dns_zone_fqdn} ${var.force_new_cert}" --verbosity=error
+      sudo /concourse/scripts/start.sh ${var.hostname}.${var.dns_zone_fqdn} ${var.force_new_cert} ;\
+      // cron tab runs every weekend e.g. on Sunday at midnight ;\
+      ! grep -q '0 0 * * 0 /concourse/scripts/prune_workers.sh' && echo '0 0 * * 0 /concourse/scripts/prune_workers.sh' | sudo tee -a /etc/crontab" --verbosity=error
```

Comment:
It could be nice to introduce a variable for the path /concourse/scripts/prune_workers.sh.

Reply:
created a symbolic link to cron.weekly, the script path is not repeating twice :)